recipes/dpkt/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "dpkt" %}
+{% set version = "1.9.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 52a92ecd5ca04d5bd852bb11cb2eac4bbe38b42a7c472e0d950eeb9f82a81e54
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - dpkt
+
+about:
+  home: http://github.com/kbandla/dpkt
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'fast, simple packet creation / parsing, with definitions for the basic TCP/IP protocols'
+  description: |
+    dpkt is a simple packet creation / parsing, with definitions for the basic TCP/IP protocols
+    and is implemented in pure python
+  doc_url: https://dpkt.readthedocs.io
+  dev_url: http://github.com/kbandla/dpkt
+
+extra:
+  recipe-maintainers:
+    - brifordwylie
+    - obormot
+    - kylekeppler
```

Comment:
@brifordwylie and @obormot need to confirm they are OK being maintainers

Reply:
@brifordwylie, @obormot, you cool with being listed here?

Reply:
I'm an active maintainer of dpkt itself, but I'm not using conda personally so listing me as a *recipe* maintainer here would be a stretch..  I'd rather not.  Thanks for checking!

Reply:
Great. @kylekeppler can you please remove @obormot from the list?